source/_components/calendar.todoist.markdown

```diff
@@ -0,0 +1,136 @@
+---
+layout: page
+title: "Todoist"
+description: "Instructions how to integrate Todoist into Home Assistant."
```

Comment:
**Spelling & Grammar**:
- `Instructions how to integrate` -> `Instructions on how to integrate`